### PR TITLE
[ko] Korean translation for 3-transport lessons 1

### PR DESCRIPTION
**Pull request by WZNT-KimHyeJin**
### create README.ko.md

### CODE REVIEW

**Comment by WZNT-KimHyeJin** [comment]
@DobiIsFree @jhYun505  It's your turn!! I need your perfect review

File: 3-transport/lessons/1-location-tracking/translations/README.ko.md [hunk 1]
```diff
@@ -10,195 +10,196 @@
...
 ---
 
-## 🚀 Challenge
+## 🚀 도전
 
-Write your own NMEA decoder! Rather than relying on third party libraries to decode NMEA sentences, can you write your own decoder to extract latitude and longitude from NMEA sentences?
+여러분만의 NMEA 디코더를 쓰세요! NMEA 문장을 해독하기 위해 타사 라이브러리에 의존하는 대신, 여러분은 NMEA 문장에서 위도와 경도를 추출하기 위해 여러분만의 디코더를 작성할 수 있을까요?
 
-## Post-lecture quiz
+## 가의 후 퀴즈
```
Comment (DobiIsFree) [inline 3-transport/lessons/1-location-tracking/translations/README.ko.md:193]: I found typo.😉

Comment (jimbobbennett) [reply 3-transport/lessons/1-location-tracking/translations/README.ko.md:193]: Does this need to be resolved? If so, I'll merge after it is fixed.

Comment (WZNT-KimHyeJin) [reply 3-transport/lessons/1-location-tracking/translations/README.ko.md]: Yes, it has to be resolved. And I fixed the typo. So you can merge it!

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**